.github/workflows/e2e.yml

```diff
@@ -0,0 +1,40 @@
+name: Soroban Tools e2e
+
+on:
+  push:
+    branches: [main, release/**]
+  pull_request:
+
+jobs:
+
+  integration:
+    name: System tests
+    strategy:
+      matrix:
+        scenario-filter: ["^TestDappDevelop$/^.*$"]
+    runs-on: ubuntu-latest
+    env:
+      SYSTEM_TEST_DOCKER_REPO_TAG: "sreuland/system-test:latest" 
```

Comment:
this is just short term to check that workflow works, will be changing this to `stellar/system-test:latest` shortly

Reply:
removed, use stellars repo now